dvc/command/imp.py

```diff
@@ -62,4 +64,14 @@ def add_parser(subparsers, parent_parser):
     import_parser.add_argument(
         "out", nargs="?", help="Destination path to put files to."
     )
+    import_parser.add_argument(
+        "-f",
+        "--file",
+        help="Specify name of the stage file. It should be "
+        "either 'Dvcfile' or have a '.dvc' suffix (e.g. "
+        "'prepare.dvc', 'clean.dvc', etc) in order for "
+        "dvc to be able to find it later. By default "
+        "the first output basename + .dvc is used as "
+        "a stage filename.",
```

Comment:
1. Do we enforce the file name somewhere? What would happen if I specify it w/o `.dvc`? We need to check that it does not allow creating stage files like this. Also, let's add a test for this.
2. There is not first or second output in the import case.

In general, I don't like this description. It's too verbose. I would just keep something like - `specify the name of the DVC file it generates`. All other details should go to the docs.

The same questions stay for the `dvc add`. As we agreed in private we do both (even though it's turned out `-f` is already supported by `dvc add` but not documented).